### ISSUE TITLE
Implement historical information from view

### ISSUE THREAD

**Issue by AndrewM131** [uchicago-cs/chigame#300]
This will allow for the historical information of a user to be displayed on a page. For example, how many games a user has won, and achievements of that degree.

**Comment by AndrewM131** [comment]
Update: PR has been made, but now review yet

**Comment by AndrewM131** [comment]
Update: I implemented the code such that there is statistics for matches played, matches won, tournaments played, tournaments won. I added the attributes in views.py in the users folder. I also went into userprofile_detail.html and added a link to the user's stats. The button works and goes to the user's stats. This can be accessed by any user if they go on the user's profile.

**Comment by AndrewM131** [comment]
Closing comment: In this issue, we successfully created a view that can be accessed in the profile of a user that displays the stats of a user, so for example, how many matches a user has been in, how many matches they've won, how many tournaments they've participated in and how many have they won. This is visible to any user when looking at another user's profile. In the future, there should be something that mandates a lobby for a match. Also in the future, there should be more stats, such as rank in Chigame website, and maybe win rate, and things like that.
We met twice to discuss progress and updates on the issue, and during these meetings, I coded on my laptop with comments from Henrietta and Ishi. We also communicated over slack if we made progress on the issue, and Rishabh reviewed and help fix issues with the merge.

**Comment by AndrewM131** [comment]
Closing comment: In this issue, we successfully created a view that can be accessed in the profile of a user that displays the stats of a user, so for example, how many matches a user has been in, how many matches they've won, how many tournaments they've participated in and how many have they won. This is visible to any user when looking at another user's profile. One thing is that match count is linked to lobby, and since an admin can manually add a user to a match without a lobby, there should be something in the future that mandates a lobby for a match such that the numbers stay consistent. Also in the future, there should be more stats, such as rank in Chigame website, and maybe win rate, and things like that.
We met over the course of the sprint 2 or 3 times to code, mainly on my laptop, and Henrietta and Ishi would review and offer suggestions on the code. We also communicated over slack of any progress we made. Rishabh reviewed and fixed the finishing issues with the PR.
